config/redirects.yml

```
@@ -218,16 +218,16 @@ redirects:
   "/five-things-every-prospective-teacher-needs-to-know": "/steps-to-become-a-teacher"
   "/explore-my-options/return-to-teaching/return-to-teaching-in-england-from-overseas": "/non-uk-teachers/return-to-england-after-teaching-overseas"
   "/submitting-your-application": "/how-to-apply-for-teacher-training"
-  "/competitive-salary-and-great-benefits": "/salaries-and-benefits"
-  "/explore-my-options/further-education-teacher-training/bursaries": "/become-a-further-education-teacher"
+  "/competitive-salary-and-great-benefits": "/is-teaching-right-for-me/teacher-pay-and-benefits"
+  "/explore-my-options/further-education-teacher-training/bursaries": "/is-teaching-right-for-me/become-a-further-education-teacher"
   "/explore-my-options/teach-languages": "/train-to-be-a-teacher"
   "/explore-my-options/teach-primary-as-a-maths-specialist": "/train-to-be-a-teacher"
   "/explore-my-options/teach-primary-as-a-pe-specialist": "/train-to-be-a-teacher"
   "/explore-my-options/teach-primary": "/train-to-be-a-teacher"
   "/explore-my-options/teacher-trai-ning-routes/specialist-training-options/researchers-in-schools": "/train-to-be-a-teacher"
   "/explore-my-options/teacher-training-routes/specialist-training-options/teach-first": "/train-to-be-a-teacher"
   "/why-teach": "/blog"
-  "/become-an-early-years-teacher": "/early-years-teaching-training"
+  "/become-an-early-years-teacher": "/is-teaching-right-for-me/early-years-teaching-training"
```

```suggestion
  "/become-an-early-years-teacher": "/is-teaching-right-for-me/early-years-teacher-training"
```

Since we are renaming this URL anyway, could we correct it to this? Obviously the file itself will need renaming as well - and the rest of the redirects will need changing!